README.md

```diff
@@ -2,9 +2,9 @@
 
 # smtp-client
 
-> Simple, promisified, protocol-based SMTP client for Node.js.
+> Simple promised, protocol-based SMTP client for Node.js.
```

Comment:
Promisify is a method that defines in utilities module of Node. js the standard library, so "promisified" is correct.